### ISSUE TITLE
No weight decay

### ISSUE THREAD

**Issue by nharada1** [raghakot/keras-resnet#25]
I notice this implementation does not apply weight decay as in the original paper. Is this an oversight or did you find that weight decay was not required for good results?

**Comment by ahundt** [comment]
I've been having my training stall out around 84% in that cifar10 script,
I'm thinking weight decay and reducing the learning rate after a while
would likely help.

**Comment by nharada1** [comment]
Using a learning rate schedule is absolutely critical to good performance with plain old SGD. I definitely do recommend that. 

For the weight decay, let me know if it helps. The original AlexNet paper states that weight decay was vital for learning, but it's unclear if that applies to this network as well.

**Comment by rkchee** [comment]
I am noticing the same thing.  Has anyone been able to move the 84%.

**Comment by raghakot** [comment]
I am looking into this.

**Comment by ahundt** [comment]
looks like a straightforward parameter https://keras.io/regularizers/
